pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -7,15 +7,27 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/allowedcmd"
 	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
 	"github.com/osquery/osquery-go/plugin/table"
 )
 
 // allowedCommand encapsulates the possible binary path(s) of a command allowed to execute
 // along with a strict list of arguments.
 type allowedCommand struct {
-	binPaths []string
-	args     []string
+	bin  allowedcmd.AllowedCommand
+	args []string
+}
+
+var allowedCommands = map[string]allowedCommand{
```

Comment:
Unifying this works today, but may not work in the future -- I sorta expect we'll have totally different command here. (Not just different paths)